orchestrator-salt/src/main/resources/salt-common/salt/fluent/settings.sls

```diff
@@ -84,7 +84,7 @@
 {% set cloudera_public_gem_repo = 'https://repository.cloudera.com/cloudera/api/gems/cloudera-gems/' %}
 {% set cloudera_azure_plugin_version = '1.0.1' %}
 {% set cloudera_azure_gen2_plugin_version = '0.3.1' %}
-{% set cloudera_databus_plugin_version = '1.0.4' %}
+{% set cloudera_databus_plugin_version = '1.0.5' %}
```

Comment:
For this we will need a new image or it will break the no-internet test again. This will run on a regular highstate.

Reply:
yes we will need new images for 2.26